docs/getting-started/publishing-keys.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+---
+
+# Key Publishing Guide
+
+This page explains how to configure the server and application to allow
+publishing keys.
+
+## Prerequisites
+
+If you haven't started developing a mobile app, please take a look at the
+[Android Exposure Notifications Reference App](https://github.com/google/exposure-notifications-android)
+
+You will need to
+[complete the necessary steps](https://support.google.com/googleplay/android-developer/contact/expo_notif_api)
+to use the exposure notifcations API. You should also ensure that you have
+taken the steps to comply with the
+[Terms of Service](https://blog.google/documents/72/Exposure_Notifications_Service_Additional_Terms.pdf).
+For instance, your app may not request any of the following permissions:
+  * ACCESS_FINE_LOCATION
+  * ACCESS_COARSE_LOCATION
+  * ACCESS_BACKGROUND_LOCATION
+  * BLUETOOTH_ADMIN
+
+
+## Publishing Keys
```

Comment:
we should say something about region here.

it's rather undefined as far as we are concerned- the server operator needs to agree upon region designations w/ the app and public heath authority